driver-sync/src/main/com/mongodb/client/internal/MongoClientImpl.java

```diff
@@ -48,26 +49,29 @@
 import java.util.List;
 
 import static com.mongodb.assertions.Assertions.notNull;
-import static org.bson.internal.CodecRegistryHelper.createRegistry;
 import static com.mongodb.client.internal.Crypts.createCrypt;
 import static com.mongodb.internal.event.EventListenerHelper.getCommandListener;
+import static org.bson.internal.CodecRegistryHelper.createRegistry;
 
 public final class MongoClientImpl implements MongoClient {
 
     private final MongoClientSettings settings;
+    private final MongoDriverInformation mongoDriverInformation;
```

Comment:
This is really just here for unit testing, to ensure that it was constructed properly